### PR TITLE
[Event] Drop `transaction_engine_v2_at` and `pending_transaction_engine_at` columns part 2/3

### PR DESCRIPTION
**Pull request by rluodev**
this part: drop columns

next part: unignore columns

### CODE REVIEW

File: app/models/event.rb [hunk 1]
```diff
@@ -20,7 +20,6 @@
 #  donation_reply_to_email                      :text
 #  donation_thank_you_message                   :text
 #  expected_budget                              :integer
-#  has_fiscal_sponsorship_document              :boolean
```
Comment (garyhtou) [inline app/models/event.rb]: Please drop unrelated changes. ty!

Comment (rluodev) [reply app/models/event.rb]: fixed!

**Review by garyhtou — APPROVED** [review]
amazing
